src/Symfony/Component/Security/Http/Authenticator/RememberMeAuthenticator.php

```diff
@@ -88,9 +89,21 @@ public function authenticate(Request $request): Passport
             throw new \LogicException('No remember-me cookie is found.');
         }
 
-        $rememberMeCookie = RememberMeDetails::fromRawCookie($rawCookie);
+        if (!\is_callable([$this->rememberMeHandler, 'getRememberMeDetails'])) {
+            trigger_deprecation('symfony/security-http', '6.1', 'RememberMeHandlers should implement method "getRememberMeDetails()".');
```

Comment:
```suggestion
```

I think we can remove the deprecation here - the DebugClassLoader already triggers a similar deprecation based on the `@method` PHPdoc you added to the interface.

Reply:
If the deprecations are already triggered that way then I agree these would be unnecessary. Changes made.